xapian-bindings/python3/docs/examples.rst

```diff
@@ -41,3 +41,33 @@ simplematchdecider.py
    :language: python
    :emphasize-lines: 12,15-18
    :linenos:
+
+.. _xapian-prepare-trainingfile:
+
+xapian-prepare-trainingfile.py
+=====================
+
+.. literalinclude :: examples/xapian-prepare-trainingfile.py
+   :language: python
+   :emphasize-lines: 12,15-18
```

Comment:
These lines are meant to be the lines in the example code which are the interesting part of the code, not just copied verbatim from an existing example (though they also seem to be wrong for all the existing examples - we're currently emphasising some lines in the licensing boilerplate).

Reply:
Having looked at this a bit, I think it's best just to drop this idea - it's only really visible in the output if you know it's there (https://xapian.org/docs/bindings/python3/examples.html) and will require updating each time the example code changes in a way which adds or removes lines.  If it actually improved the output visibly then we could consider how to automate that, but given you can't really see the effect that doesn't seem worth doing.

Reply:
fixed in a fixup commit.